### ISSUE TITLE
revisar maestria pala heal

### ISSUE THREAD

**Issue by 50megas** [RoDNet/Bugtracker-Infinium-4.0.6a#617]
la maestria  http://es.wowhead.com/spell=76669 pare que hora no funcina,le pregunte a un pala y se curo con todas las curas, istantaneas y de casteo  y no le saltaba ningun escudo

**Comment by crowel** [comment]
Hola yo soy ese pala, el escudo no salta ni curandome a mi ni salta en la persona que estoy curando, al menos ahora no sale en los dos a la vez jajaja, echenle un vistazo cuando tengan un momento pls, graciass.

**Comment by searum** [comment]
no, ahora esta desactivado porque lo arregle pero crasheaba... tengo que mirarlo bien :)

**Comment by crowel** [comment]
Ah perdona queda en tus manos :), un saludo
